docs/.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.4.2
```

Comment:
Using asdf to build asdf.

Reply:
Yes! I have this nested as to not require `nodejs` when developing the core code.